website/pages/docs/commands/operator/raft.mdx

```diff
@@ -0,0 +1,154 @@
+
+```text
+Usage: vault operator raft join [options] <leader-api-addr>
+
+  Join the current node as a peer to the raft cluster by providing the address
+  of the raft leader node.
+
+	  $ vault operator raft join "http://127.0.0.2:8200"
+```
+
+### Parameters
+
+The following flags are available for the `operator raft join` command.
+
+- `-leader-ca-cert` `(string: "")` - CA cert to communicate with raft leader.
+
+- `-leader-client-cert` `(string: "")` - Client cert to to authenticate to raft leader.
+
+- `-leader-client-key` `(string: "")` - Client key to to authenticate to raft leader.
+
+- `-non-voter` `(bool: false) (enterprise)` - This flag is used to make the
+  server not participate in the Raft quorum, and have it only receive the data
+  replication stream. This can be used to add read scalability to a cluster in
+  cases where a high volume of reads to servers are needed. The default is false.
+
+- `-retry` `(bool: false)` - Continuously retry joining the raft cluster upon
+  failures. The default is false.
+
+## list-peers
+
+This command is used to list the full set of peers in the raft cluster.
+
+```text
+Usage: vault operator raft list-peers
+
+  Provides the details of all the peers in the raft cluster.
+
+	  $ vault operator raft list-peers
+```
+
+### Example Output
+
+```python
+{
+ ...
+  "data": {
+    "config": {
+      "index": 62,
+      "servers": [
+        {
+          "address": "127.0.0.2:8201",
+          "leader": true,
+          "node_id": "node1",
+          "protocol_version": "3",
+          "voter": true
+        },
+        {
+          "address": "127.0.0.4:8201",
+          "leader": false,
+          "node_id": "node3",
+          "protocol_version": "3",
+          "voter": true
+        }
+      ]
+    }
+  }
+}
+```
+
+## remove-peer
+
+This command is used to remove a node from being a peer to the raft cluster. In
+certain cases where a peer may be left behind in the Raft configuration even
+though the server is no longer present and known to the cluster, this command
+can be used to remove the failed server so that it is no longer affects the Raft
+quorum.
+
+```text
+Usage: vault operator raft remove-peer <server_id>
+
+  Removes a node from the raft cluster.
+
+	  $ vault operator raft remove-peer node1
+```
+
+## snapshot
+
+This command groups subcommands for operators interacting with the snapshot
+functionality of the raft storage backend. There are 2 subcommands supported:
+`save` and `restore`.
+
+```text
+Usage: vault operator raft snapshot <subcommand> [options] [args]
+
+  This command groups subcommands for operators interacting with the snapshot
+  functionality of the raft storage backend.
+
+Subcommands:
+    restore    Installs the provided snapshot, returning the cluster to the state defined in it
+    save       Saves a snapshot of the current state of the raft cluster into a file
+```
+
+### snapshot save
+
+Takes a snapshot of the raft cluster state.
```

Comment:
Is this just the Raft cluster state, or is it the entirety of the storage backend? I think we need to clarify.

Reply:
It takes backup of the storage. Clarified.